xarray/backends/zarr.py

```diff
@@ -3,6 +3,8 @@
 
 import numpy as np
 
+from xarray.tests import LooseVersion
```

Comment:
this is the reason for the failures of `pint`:
```suggestion
from distutils.version import LooseVersion
```
the reason is that `pint` tries to import `xarray`, but we try to import it in `xarray.tests.test_units`, so we get a circular dependency graph. See also #4751.

Also, `xarray.tests.LooseVersion` assumes the old versioneer versions, but we switched to `setuptools-scm` some time ago which produces something like `0.16.3.dev114+gb848c66d.d20210119`

Reply:
Thanks!
The version in question is for zarr, which uses the old typical style of numbers.